plugins/BEdita/Core/src/Model/Action/ListRelatedFoldersAction.php

```diff
@@ -36,4 +39,25 @@ public function execute(array $data = [])
 
         return $result;
     }
+
+    /**
+     * Remove unnecessary fields from `Tree` entity.
+     *
+     * @param \Cake\Datasource\EntityInterface $joinData Join data entity.
+     * @return void
+     */
+    protected function prepareJoinEntity(EntityInterface $joinData): void
+    {
+        if ($joinData instanceof Tree) {
```

Comment:
I was thinking if it wasn't better to remove property in `Folder` entity overriding `ObjectEntity::getMeta()`.
Maybe something as:

```php
protected function getMeta()
{
    return array_diff_key(parent::getMeta(), array_flip(['id', 'parent_id', 'object_id', 'root_id', 'parent_node_id', 'tree_left', 'tree_right']));
}
```